server/text_generation_server/models/custom_modeling/flash_mistral_modeling.py

```diff
@@ -39,8 +36,14 @@
     TensorParallelHead,
     get_linear,
 )
+from text_generation_server.utils.import_utils import IS_CUDA_SYSTEM, IS_ROCM_SYSTEM
+
+if IS_CUDA_SYSTEM:
+    import dropout_layer_norm
+elif IS_ROCM_SYSTEM:
+    from vllm import layernorm_ops
```

Comment:
This kind of begs for a proper `enum` and `match`.

Too bad python is not rust :)

Reply:
https://peps.python.org/pep-0635/